package.json

```diff
@@ -48,6 +49,7 @@
     "typescript": "4.3.4",
     "uuid": "^8.3.2",
     "web3": "^1.6.1",
+    "wget-improved": "^3.3.0",
```

Comment:
If possible, I would like to remove this dependency. You can see my note at where you use it for more details.

Reply:
removed this in recent commit